### PR TITLE
BPTL Dashboard: UI for home collection Quick Fix (Part 2.4)

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
https://github.com/episphere/connect/issues/771

This PR addresses following issues:
 -  Kit Assembly Page: fix bug (manual zeros)
 - Assign Kit Page: check front and back end(firestore) for kits that have already been assigned is updated
 -  Derived Variables: two variables were storing the data as a string
 -  Receipt Scenario when no card is assigned on the Kit Receipt Page set to yes/no CIDs
 - Update CID to 8583443674
 
 Related PR: https://github.com/episphere/connectFaas/pull/466

### CODE REVIEW

**Review by ramavavi — APPROVED** [review]

**Comment by ramavavi** [comment]
looks good to me.

**Review by Gbarra9 — APPROVED** [review]
Looks good

**Review by ramavavi — APPROVED** [review]

File: src/pages/homeCollection/kitsReceipt.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ export const confirmKitReceipt = () => {
           if (option.selected) {packageConditions.push(option.value)}
         }
         kitObj[`${conceptIds.pkgReceiptConditions}`] = packageConditions;
```
Comment (we-ai) [inline src/pages/homeCollection/kitsReceipt.js:169]:
```suggestion
        kitObj[conceptIds.pkgReceiptConditions] = packageConditions;
```

File: src/pages/homeCollection/kitsReceipt.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ export const confirmKitReceipt = () => {
...
-      //  kitObj[conceptIds.pkgComments] = document.getElementById('receivePackageComments').value.trim();
+        kitObj[conceptIds.pkgComments] = document.getElementById('receivePackageComments').value.trim();
```
Comment (we-ai) [inline src/pages/homeCollection/kitsReceipt.js:170]: May need `null` handling.

**Review by we-ai — APPROVED** [review]
Looks good. Minor comments.

**Review by jhflorey — APPROVED** [review]

**Review by jhflorey — APPROVED** [review]

**Pull request merged**